utils/src/treev2/quickPickWizard/experiences/contextValueExperience.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../../index';
+import * as vscode from 'vscode';
+import { RecursiveQuickPickStep } from '../RecursiveQuickPickStep';
+import { getLastNode } from '../QuickPickWizardContext';
+import { NoResourceFoundError } from '../../../errors';
+import { AzureWizardPromptStep } from '../../../wizard/AzureWizardPromptStep';
+import { AzureWizard } from '../../../wizard/AzureWizard';
+import { isWrapper } from '../../../registerCommandWithTreeNodeUnwrapping';
+import { ResourceGroupsItem } from '../quickPickAzureResource/tempTypes';
+
+export async function contextValueExperience<TPick extends unknown>(context: types.IActionContext, tdp: vscode.TreeDataProvider<ResourceGroupsItem>, contextValueFilter: types.ContextValueFilter): Promise<TPick> {
+    const promptSteps: AzureWizardPromptStep<types.QuickPickWizardContext>[] = [

```

Comment:
I'm a little bit confused here. Do we not have to still do the first three steps of `appResourceExperience` to bypass the subscription, group, and app resource?

Will `RecursiveQuickPickStep` be able to do that on its own?

Reply:
`contextValueExperience` was designed with not-necessarily-the-Azure-views in mind (e.g., I wanna use it in the Docker extension). As long as we have properly defined `isAncestorPick` / `isFinalPick`, then it will work in the Azure Resource views _and_ elsewhere.